sphinx_autobuild/__main__.py

```diff
@@ -98,7 +98,7 @@ def _parse_args(argv):
 
     # Copy needed settings
     args.sourcedir = Path(sphinx_args.sourcedir).resolve(strict=True)
-    args.outdir = Path(sphinx_args.outputdir).resolve(strict=True)
+    args.outdir = Path(sphinx_args.outputdir).resolve()
     if sphinx_args.doctreedir:
         args.doctree_dir = Path(sphinx_args.doctreedir).resolve(strict=True)
```

Comment:
```suggestion
        args.doctree_dir = Path(sphinx_args.doctreedir).resolve()
```

Had some time to test, and your fix fails for my build options - but probably due to the fact I specify a specific directory for doctrees - which is also currently strict.

Error log here:

```
. .sphinx/venv/bin/activate; .sphinx/venv/bin/sphinx-autobuild -b dirhtml "." "_build" -c . -d .sphinx/.doctrees -j auto
Traceback (most recent call last):
  File "/home/michael/Documents/Code/sphinx-docs-starter-pack/sp-docs/.sphinx/venv/bin/sphinx-autobuild", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/home/michael/Documents/Code/sphinx-docs-starter-pack/sp-docs/.sphinx/venv/lib/python3.12/site-packages/sphinx_autobuild/__main__.py", line 34, in main
    args, build_args = _parse_args(list(argv))
                       ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/michael/Documents/Code/sphinx-docs-starter-pack/sp-docs/.sphinx/venv/lib/python3.12/site-packages/sphinx_autobuild/__main__.py", line 103, in _parse_args
    args.doctree_dir = Path(sphinx_args.doctreedir).resolve(strict=True)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/michael/.asdf/installs/python/3.12.4/lib/python3.12/pathlib.py", line 1240, in resolve
    s = self._flavour.realpath(self, strict=strict)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen posixpath>", line 427, in realpath
  File "<frozen posixpath>", line 462, in _joinrealpath
FileNotFoundError: [Errno 2] No such file or directory: '../sp-files/.sphinx/.doctrees'
make[1]: *** [Makefile.sp:61: sp-run] Error 1
make[1]: Leaving directory '/home/michael/Documents/Code/sphinx-docs-starter-pack/sp-docs'
make: *** [Makefile:30: run] Error 2

```

With your changes and mine combined, the output is error free.